src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsFeaturedVideoSoundOasis.java

```diff
@@ -0,0 +1,37 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.dataprovider.ads.AdsDataProvider;
+import com.wikia.webdriver.common.templates.TemplateNoFirstLoad;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsJWPlayerObject;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class TestAdsFeaturedVideoSoundOasis extends TemplateNoFirstLoad {
+
+  private static final Duration AD_LENGTH = Duration.ofSeconds(30);
+
+  @Test(
+      groups = {"AdsFeaturedVideoSoundOasis", "AdsJWPlayerPrerollOasisWithSound"}
```

Comment:
Do we need two groups?

Reply:
IMO, yes, because if you need to run only one function, it's possible only by this. But I'm not sure if it is used right now (except development)